trieste/acquisition/function/active_learning.py

```diff
@@ -222,7 +222,7 @@ def acquisition(x: TensorType) -> TensorType:
         t = (threshold - mean) / stdev
         t_plus = t + alpha
         t_minus = t - alpha
-        normal = tfp.distributions.Normal(tf.cast(0, x.dtype), tf.cast(1, x.dtype))
+        normal = tfp.distributions.Normal(tf.constant(0, x.dtype), tf.constant(1, x.dtype))
```

Comment:
why is using constant better than casting?

Reply:
tf.cast should be used for Tensors, tf.constant for static, non-Tensor types, and tf.convert_to_tensor for any mix of Tensor-like objects. Also, it reads better here, and makes testing for unnecessary casts easier.

Reply:
thanks for the education :)